src/loops/programs_with_repetition/medium/q4/OurGlass.java

```diff
@@ -0,0 +1,35 @@
+package loops.programs_with_repetition.medium.q4;
+
+/*
+ * Write a program to print the following pattern using nested loops : 
```

Comment:
This question does not work as a java question because they could just hardcode the output.

Reply:
Either the question needs to be changed to take an n to vary the size of the shape, or to a parsons question.